code/src/reducers/todos.js

```diff
@@ -0,0 +1,31 @@
+import { createSlice } from "@reduxjs/toolkit"
+
+const todos = createSlice({
+  name: "todos",
+  initialState: {
+    items: [],
```

Comment:
Good that you used an empty array (so that you removed the hard-coded code used for developing).